### ISSUE TITLE
Cache more headers

### ISSUE THREAD

**Issue by brong** [cyrusimap/cyrus-imapd#1596]
From: [Bron Gondwana](brong@fastmail.fm)
Bugzilla-Id: 3736
Version: 2.4.16
Owner: [Bron Gondwana](brong@fastmail.fm)

**Comment by ajaysusarla** [comment]
All the Apple `x-headers` are cached now(fixed in this commit c70c950cd966fbe8039ab457260699f29f0712c5). Do we want to cache thes headers?:
```
    { "bcc", BIT32_MAX },
    { "cc", BIT32_MAX },
    { "date", BIT32_MAX },
    { "delivery-date", BIT32_MAX },
    { "envelope-to", BIT32_MAX },
    { "from", BIT32_MAX },
    { "in-reply-to", BIT32_MAX },
    { "mime-version", BIT32_MAX },
    { "reply-to", BIT32_MAX },
    { "sender", BIT32_MAX },
    { "subject", BIT32_MAX },
    { "to", BIT32_MAX },
```

**Comment by ajaysusarla** [comment]
Brong mentioned that these headers:
```
    { "bcc", BIT32_MAX },
    { "cc", BIT32_MAX },
    { "date", BIT32_MAX },
    { "delivery-date", BIT32_MAX },
    { "envelope-to", BIT32_MAX },
    { "from", BIT32_MAX },
    { "in-reply-to", BIT32_MAX },
    { "mime-version", BIT32_MAX },
    { "reply-to", BIT32_MAX },
    { "sender", BIT32_MAX },
    { "subject", BIT32_MAX },
    { "to", BIT32_MAX },
``` 
are already in the envelope, so we shouldn't be caching them again here. 
Closing this issue, as the reported issue is actually fixed.
